sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/models/SentenceSentiment.java

```diff
@@ -77,12 +97,20 @@ public TextSentiment getSentiment() {
     }
 
     /**
-     * Get the confidence score of the sentiment label. All score values sum up to 1, the higher the score, the
-     * higher the confidence in the sentiment.
+     * Get the offset of sentence. The start position for the sentence in a document.
      *
-     * @return The {@link SentimentConfidenceScores}.
+     * @return The offset of sentence.
      */
-    public SentimentConfidenceScores getConfidenceScores() {
-        return confidenceScores;
+    public int getOffset() {
+        return offset;
+    }
+
+    /**
+     * Get the length of sentence.
+     *
+     * @return The length of sentence.
+     */
+    public int getLength() {
+        return length;
```

Comment:
Can you add tests for the new `offset` and `length` properties?

Reply:
All tests that verifying the `SentenceSentiment` has verifying the offset and length. See https://github.com/Azure/azure-sdk-for-java/pull/14599/files/8f914029e29ac9d387803cc94f8db120ed7fcaba#diff-6152e7741d610810c960c85cfc56fbc2R361

Reply:
Oh I'm so sorry I don't know how I totally missed those tests

Reply:
No worries. Thank you for reviewing the PR.